Doc/library/decimal.rst

```diff
@@ -238,6 +240,23 @@ floating-point flying circus:
    >>> c % a
    Decimal('0.77')
 
+Decimals can be formatted in fixed-point or scientific notation, using the same
+formatting syntax (see :ref:`formatspec`) as builtin :class:`float` type:
+
+.. doctest::
+
+   >>> format(Decimal('2.675'), "f")
+   '2.675'
+   >>> format(Decimal('2.675'), ".2f")
+   '2.68'
+   >>> format(Decimal('2.675'), ".2e")
+   '2.68e+0'
+   >>> with localcontext() as ctx:
+   ...     ctx.rounding = ROUND_DOWN
+   ...     print(format(Decimal('2.675'), ".2f"))
+   ...
```

Comment:
I think it would be worthwhile to mention f-strings and `str.format()` here. I don't think many people are familiar with `format`, and don't realize that all 3 use `obj.__format__` behind the scenes.